File: web/src/api/sysLoginLog.js
import service from '@/utils/request'

export const deleteLoginLog = (data) => {
  return service({
    url: '/sysLoginLog/deleteLoginLog',
    method: 'delete',
    data
  })
}

export const deleteLoginLogByIds = (data) => {
  return service({
    url: '/sysLoginLog/deleteLoginLogByIds',
    method: 'delete',
    data
  })
}

export const getLoginLogList = (params) => {
  return service({
    url: '/sysLoginLog/getLoginLogList',
    method: 'get',
    params
  })
}

export const findLoginLog = (params) => {
    return service({
        url: '/sysLoginLog/findLoginLog',
        method: 'get',
        params
    })
}

File: server/router/system/sys_login_log.go
package system

import (
	"github.com/flipped-aurora/gin-vue-admin/server/api/v1"
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type LoginLogRouter struct{}

func (s *LoginLogRouter) InitLoginLogRouter(Router *gin.RouterGroup) {
	loginLogRouter := Router.Group("sysLoginLog").Use(middleware.OperationRecord())
	loginLogRouterWithoutRecord := Router.Group("sysLoginLog")
	sysLoginLogApi := v1.ApiGroupApp.SystemApiGroup.LoginLogApi
	{
		loginLogRouter.DELETE("deleteLoginLog", sysLoginLogApi.DeleteLoginLog)           // 删除登录日志
		loginLogRouter.DELETE("deleteLoginLogByIds", sysLoginLogApi.DeleteLoginLogByIds) // 批量删除登录日志
	}
	{
		loginLogRouterWithoutRecord.GET("findLoginLog", sysLoginLogApi.FindLoginLog)       // 根据ID获取登录日志(详情)
		loginLogRouterWithoutRecord.GET("getLoginLogList", sysLoginLogApi.GetLoginLogList) // 获取登录日志列表
	}
}
